daphne/src/roles_test.rs

```diff
@@ -7,34 +7,32 @@ use crate::{
         MEDIA_TYPE_AGG_CONT_REQ, MEDIA_TYPE_AGG_INIT_REQ, MEDIA_TYPE_AGG_SHARE_REQ,
```

Comment:
Reviewer note: The bulk of the changes here are due to using `Test` to drive tests instead of `MockAggregator`. The test cases should not have changed.